ios/FluentUI.Demo/FluentUI.Demo/Demos/LeftNavDemoController.swift

```diff
@@ -46,7 +60,7 @@ class LeftNavDemoController: DemoController, MSFDrawerControllerDelegate {
         self.navigationController?.interactivePopGestureRecognizer?.isEnabled = true
     }
 
-    private var drawerController: MSFDrawer!
+    private var drawerController: DrawerController!
```

Comment:
Can we make this a lazy var and put the contents of lines 43 to 50 in its initialization?
The navigationButton will likely need to be a lazy var as well.